server-example/requirements.txt

```diff
@@ -1,2 +1,3 @@
 Flask>1.1
 pybadges
+pytest
```

Comment:
Hmmmm....maybe it would be better to have a separate requirements file for tests? Like:
requirements-test.txt
-r requirements.txt
pytest

Also, could you hook this up to the nox tests? You probably have to edit `_run_tests` and `unit` in noxfile.py